bridge/client/app/_interceptors/http-error-interceptor.spec.ts

```diff
@@ -118,4 +118,86 @@ describe('HttpErrorInterceptorService', () => {
     // then
     expect(spy).toHaveBeenCalledTimes(0);
   });
+
+  it('should show error only once in case of 401 and OAUTH', async () => {
+    // given
+    const spy = jest.spyOn(TestBed.inject(NotificationsService), 'addNotification');
+
+    await apiService.getMetadata().subscribe();
```

Comment:
await actually does not have any effect here because `subscribe()` does not return a promise.
`await apiService.getMetadata().pipe(take(1)).toPromise()` would work here. Please adapt it for all occurencies.

Reply:
Actually the `await` is not necessary, so I removed the `await` and `async` from the tests.